config.go

```diff
@@ -591,6 +594,13 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
+	// Ensure a valid max channel fee allocation was set.
+	if cfg.MaxChannelFeeAllocation <= 0 || cfg.MaxChannelFeeAllocation > 1 {
+		return nil, fmt.Errorf("invalid max channel fee allocation: "+
+			"%v, must be within [0.1, 1]",
```

Comment:
Fixed the range. Don't think we can use the constants in the description of the flag though?

Reply:
Oh yes, that is a tag. I don't think so indeed. But the range in the tag is still `[0.1, 1]` instead of `[0, 1]`. Also the tag mentions percentage which suggests it is in the 0-100 range?